### PR TITLE
Mes 4856 terminate tests throughout driving test

### PR DESCRIPTION
**Pull request by JCarterDVSA**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/features/cat-c/10-pass_with_no_faults.feature [hunk 1]
```diff
@@ -15,7 +15,7 @@ Feature: A Driving Examiner Completes a Passed User Journey in Category C
       When I open the reversing diagram
       Then I should see the reversing diagram modal
       And I close the reversing diagram modal
-      Then I close the revresing diagram drop down
+      Then I close the reversing diagram drop down
```
Comment (turnerc1) [inline test/e2e/features/cat-c/10-pass_with_no_faults.feature:18]: Change this line on every page it is used.

Comment (JCarterDVSA) [reply test/e2e/features/cat-c/10-pass_with_no_faults.feature:18]: done

File: test/e2e/features/cat-c/13-terminate.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    Then the transmission is selected
+    And I continue to the back to office page
+    Then I am on the back to office page
+    And I continue to the office write up
+    Then I should see the "Office" page
+    And the office page test outcome is "Terminated"
+    When I complete the office write up
+    And I upload the test
+    Then I should see the "Journal" page
+    And the test result for "Mr Right Ford" is "11"
```
Comment (turnerc1) [inline test/e2e/features/cat-c/13-terminate.feature:32]: Should there only be two times where you can see see if you can press the terminate button?

Comment (JCarterDVSA) [reply test/e2e/features/cat-c/13-terminate.feature:32]: no, it's a test where you actually terminate a test, there are multiple reasons to terminate a test. this is just one way to do it.

**Review by turnerc1 — APPROVED** [review]

**Review by matthew2564 — APPROVED** [review]

**Pull request merged**